doozer/doozerlib/image.py

```
@@ -1,23 +1,89 @@
+                rhel_version = isolate_el_version_in_release(release_label)
+
+        # Extract golang version from version label if we don't have it yet
+        if not golang_version:
+            version_label = labels.get('version')
+            if version_label:
+                version_fields = util.extract_version_fields(version_label, at_least=2)
+                golang_version = (version_fields[0], version_fields[1])
+
+    except Exception:
+        # Return whatever we managed to extract before the error
+        pass
```

Ruff (0.14.14)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)